### PR TITLE
add a tests to try delete rbd reffered by sc

### PR DESCRIPTION
**Pull request by DanielOsypenko**
continuing OCS-3889 added tests that verifying that RBD pools cannot be deleted from cluster when referred by a Storage class. 

- [x] UI should be added

### CODE REVIEW

**Comment by shivamdurgbuns** [comment]
@DanielOsypenko can you please run a pr check job for this?

**Comment by DanielOsypenko** [comment]
> @DanielOsypenko can you please run a pr check job for this?

sure, I want to add few things there too, put it to WIP.

**Comment by DanielOsypenko** [comment]
test run: PASSED
http://pastebin.test.redhat.com/1107568

**Comment by DanielOsypenko** [comment]
> PR validation on existing cluster
> 
> Cluster Name: dosypenk-248 Cluster Configuration: PR Test Suite: tier1 PR Test Path: tests/manage/storageclass/test_delete_rbd_pool_attached_to_sc.py::TestDeleteRbdPool::test_delete_rbd_pool_attached_to_sc_UI Additional Test Params: OCP VERSION: 4.14 OCS VERSION: 4.14 tested against branch: master
> 
> [Job FAILED](https://url.corp.redhat.com/b096e5c) (installation failed, tests not executed).

skipped due to existing BZ mark, relaunching

**Comment by DanielOsypenko** [comment]
> PR validation on existing cluster
> 
> Cluster Name: dosypenk-248 Cluster Configuration: PR Test Suite: tier1 PR Test Path: tests/manage/storageclass/test_delete_rbd_pool_attached_to_sc.py::TestDeleteRbdPool::test_delete_rbd_pool_attached_to_sc_UI Additional Test Params: OCP VERSION: 4.14 OCS VERSION: 4.14 tested against branch: master
> 
> [Job FAILED](https://url.corp.redhat.com/b1c993c) (installation failed, tests not executed).

no tier mark, adding, relaunching

File: tests/manage/storageclass/test_delete_rbd_pool_attached_to_sc.py
Comment (jilju) [inline tests/manage/storageclass/test_delete_rbd_pool_attached_to_sc.py:1]: /lgtm

**Review by jilju — APPROVED** [review]

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**